azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/async_adapter.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from .mqtt_transport import MQTTTransport
+from azure.iot.common import asyncio_compat
+
+logger = logging.getLogger(__name__)
+
+
+class MQTTAsyncTransport(MQTTTransport):
```

Comment:
>MQTTAsyncTransport [](start = 6, length = 18)

Will this `async_adapter` file have all transports ? or can this be renamed to mqtt_async_adapter ?